src/ui/zcl_abapgit_popups.clas.abap

```diff
@@ -891,7 +891,7 @@ CLASS zcl_abapgit_popups IMPLEMENTATION.
     IF sy-subrc = 1.
 * looks like the function module used does not exist on all
 * versions since 702, so show an error
-      zcx_abapgit_exception=>raise( 'Function module PB_POPUP_PACKAGE_CREATE does not exist' ).
+      zcx_abapgit_exception=>raise( 'Create package manually, FM not part of system' ).
```

Comment:
What about:
```|Your system does not support automatic creation of packages. Please, create the package { es_package_data-devclass } manually.|```

Reply:
better, updated

devclass removed from message, as it might not be known at the time